packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -552,7 +552,7 @@ class DeepLinksController extends DisposableController
           false);
 
       if (linkdata.os.contains(PlatformOS.ios)) {
-        final List<String> iosPaths = iosDomainPaths[linkdata.domain] ?? [];
+        final List<Path> iosPaths = iosDomainPaths[linkdata.domain] ?? [];
```

Comment:
remove type on LHS. There should be a lint that catches this, and the DCM check on the CI should catch this too, so maybe we have a bug. But please remove type on LHS and add it to the null case empty list.